luzer/init.lua

```diff
@@ -53,7 +53,15 @@ local function progname(argv)
 end
 
 local function Fuzz(test_one_input, custom_mutator, func_args)
-    local flags = build_flags(arg, func_args)
+    if custom_mutator ~= nil and
+       type(custom_mutator) ~= "function" then
+        error("`custom_mutator` must be a function")
```

Comment:
You may also reformat this part as the following (it looks more readable to me):
```lua
    if custom_mutator ~= nil and
       type(custom_mutator) ~= "function"
    then
        error("`custom_mutator` must be a function")
    end
```
Feel free to ignore, though.

Reply:
Updated:

```diff
--- a/luzer/init.lua
+++ b/luzer/init.lua
@@ -54,7 +54,8 @@ end
 
 local function Fuzz(test_one_input, custom_mutator, func_args)
     if custom_mutator ~= nil and
-       type(custom_mutator) ~= "function" then
+       type(custom_mutator) ~= "function"
+    then
         error("custom_mutator must be a function")
     end
     local luzer_args = func_args or {}
```